### PR TITLE
Enable shield module by default, allow for config via config.yml.

### PR DESCRIPTION
**Pull request by stooit**
For issue #6

### CODE REVIEW

File: provisioning/tasks/drush-shield.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  sudo: no
+
+- name: Enable Shield module configuration
+  command: >
+    {{ drush_path }} variable-set shield_enabled 1 -y
+    chdir={{ drupal_core_path }}
+  when: "{{ drupal_shield_enabled }} == true"
+  sudo: no
+
+- name: Disable Shield module configuration
```
Comment (gollyg) [inline provisioning/tasks/drush-shield.yml:16]: My reading of this is that the shield module won't even be enabled, so the variable won't really do anything. Is this even required?

Oh - is is if you're rebuilding without reinstalling, and I guess also if you are importing the live db it is required.

Comment (gollyg) [inline provisioning/tasks/drush-shield.yml:16]: Actually, would we not be better disabling the shield module?

Comment (gollyg) [inline provisioning/tasks/drush-shield.yml:16]: Other than that, I have tested the configuration change and it works. So happy to merge this when this is discussed.

Comment (stooit) [inline provisioning/tasks/drush-shield.yml:16]: I'm happy either way, so long as the config variables are doing the right thing it doesn't bother me if it's enabled/disabled by default

**Pull request merged**